src/mqt/bench/benchmark_generator.py

```diff
@@ -393,8 +393,6 @@ def get_benchmark(
         msg = "compiler_settings must be of type CompilerSettings or None."  # type: ignore[unreachable]
         raise ValueError(msg)
 
-    assert (compiler_settings.tket is not None) or (compiler_settings.qiskit is not None)
-
```

Comment:
Any reason why this assertion has been removed? If it's because of how the CLI sets both to `None` if none of the parameters are passed to the CLI, the suggestion from that file should fix that condition.
That should allow you to add back the assertion here.

Reply:
It's not directly related to the CLI. IMO, the assertion at this place is "wrong" since it triggers even when requesting the `alg` level, which isn't associated with a compiler.
Additionally, the assertion is essentially duplicated by more specific assertions later on (which check that the settings corresponding to the requested compiler are set). In my view, that obviates this assertion (DRY and all).

Reply:
good points! got me convinced ;-)